tests/phpunit/integration/Core/Expirables/Expirable_ItemsTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Expirable_ItemsTest
+ *
+ * @package   Google\Site_Kit\Tests\Core\Expirables
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Tests\Core\Expirables;
+
+use Google\Site_Kit\Context;
+use Google\Site_Kit\Core\Expirables\Expirable_Items;
+use Google\Site_Kit\Core\Storage\User_Options;
+use Google\Site_Kit\Tests\TestCase;
+
+class Expirable_ItemsTest extends TestCase {
+
+	/**
+	 * @var User_Options
+	 */
+	private $user_options;
+
+	/**
+	 * @var Expirable_Items
+	 */
+	private $expirable_items;
+
+	public function set_up() {
+		parent::set_up();
+
+		$user_id = $this->factory()->user->create();
+		$context = new Context( GOOGLESITEKIT_PLUGIN_MAIN_FILE );
+
+		$this->user_options    = new User_Options( $context, $user_id );
+		$this->expirable_items = new Expirable_Items( $this->user_options );
+		$this->expirable_items->register();
+	}
+
+	public function test_add() {
+		$this->assertEmpty( $this->user_options->get( Expirable_Items::OPTION ) );
+
+		$current_time  = time();
+		$expected_time = $current_time + 1000;
+		$this->expirable_items->add( 'foo', 1000 );
+		$this->assertEquals(
+			array(
+				'foo' => $current_time + 1000,
+			),
+			$this->user_options->get( Expirable_Items::OPTION )
+		);
+
```

Comment:
I think we can remove this assertion. The point of using `assertEqualsWithDelta()` below is to allow a slight discrepancy between the actual and expected timestamps to make the tests more reliable, but this assertion is doing a direct test of the values which means it's going to be less reliable. Both of the items `foo` and `bar` are tested below so we don't need this additional assertion.

```suggestion
```